index.html

```diff
@@ -10,7 +10,7 @@
                     <h1 class="hero-main-header">Image Data Resource</h1>
                     <h2 class="hero-subheader small-10 medium-10 large-10 small-offset-1 medium-offset-1 large-offset-1">Welcome to the Image Data Resource (IDR). This online, public data repository seeks to store, integrate and serve image datasets from published scientific studies.</h2>
                     <br>
-                    <a href="/webclient/userdata/?experimenter=-1" class="large button sites-button">Take a look at the data</a>
+                    <a href="/gallery/" class="large button sites-button">Take a look at the data</a>
```

Comment:
You will need to update https://github.com/IDR/idr.openmicroscopy.org/blob/master/build.sh#L8 to ignore `/gallery` in the linkcheck step